Functions.Templates/Bindings/bindings.json

```diff
@@ -1755,6 +1755,17 @@
             "enabledInTryMode": false,
             "documentation": "$content=Documentation\\Token.md",
             "filters": ["MSGraph"],
+            "AADPermissions": [
```

Comment:
I agree that, the settings are specific to input/output that get added to function.json. However anything outside of settings array applies to all the bindings and can possibly apply to all bindings. The Express AAD registration is a resource that needs to gets added specifically for MS Graph. 

The problem here is, it does not really belong in settings or outside of settings. So either we have to create a special section like related resources or find place that is closer to its behavior. The way i see it is, AAD is a resource needed only for MS Graph binding. Just like a storage account or eventhub. Would only change if MS Graph team changes SDK. So a more ideal location for it would be settings. and it also allows you take advantage of existing constructs that apply to settings in portal.